projects/netgrif-components-core/src/lib/authentication/services/methods/basic-authentication/basic-authentication.service.spec.ts

```diff
@@ -70,9 +72,10 @@ describe('BasicAuthenticationService', () => {
     });
 
     it('should decode non-latin characters', (done) => {
```

Comment:
if the `encode/decode` methods are imported from utils. They should be tested in their own test suite in utils

Reply:
Done

Reply:
why is this test still here?

Reply:
Done